Sources/Harbor/Mock/HMocker.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  HMocker.swift
+//  Harbor
+//
+//  Created by Javier Manzo on 06/11/2024.
+//
+
+import Foundation
+
+@HRequestManagerActor
+final class HMocker: Sendable {
```

Comment:
We might want to create an actor for this, maybe we can try to add and remove all mocks at the same time, and boom, race condition